src/network/graphql/__mocks__/queries.js

```diff
@@ -0,0 +1,36 @@
+/** This is a mock module of ../queries.js,
```

Comment:
can we rename the folder for `__mocks__` to just `mocks`? or is it needed to have the underscores?

Reply:
It seems to be a must. Google & Jest official tells me nothing about changing the folder name.

Reply:
does the tests work if you do rename it?

Reply:
no it does not